php_phongo.c

```diff
@@ -1199,12 +1199,12 @@ bool php_phongo_server_to_zval(zval* retval, mongoc_server_description_t* sd) /*
 
 		PHONGO_BSON_INIT_DEBUG_STATE(state);
 
-		if (!php_phongo_bson_to_zval_ex(bson_get_data(is_master), is_master->len, &state)) {
+		if (!php_phongo_bson_to_zval_ex(bson_get_data(hello_response), hello_response->len, &state)) {
 			zval_ptr_dtor(&state.zchild);
 			return false;
 		}
 
-		ADD_ASSOC_ZVAL_EX(retval, "last_is_master", &state.zchild);
+		ADD_ASSOC_ZVAL_EX(retval, "last_hello_response", &state.zchild);
```

Comment:
Since we never documented the returned fields, I decided to drop the old field entirely. Let me know if you think we should keep it in for now.

Reply:
This is just used for debug output, since Manager and Server are not serializable. No objections to changing this, since we don't document debug output at all.